cypress-tests/cypress/support/pageBuilder/pbListPages.ts

```diff
@@ -0,0 +1,52 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbListPages(data: any): Chainable<Promise<any[]>>;
+        }
+    }
+}
+
+const ERROR_FIELDS = `
```

Comment:
Feel free to paste these fields directly into L35, without using `ERROR_FIELDS`. Remove `ERROR_FIELDS` afterwards.